### PR TITLE
fix-ri-sp-without-crawler

### PR DESCRIPTION
**Pull request by iakov-aws**
*Issue #, if available:*

*Description of changes:*

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: cid/helpers/cur.py [hunk 1]
```diff
@@ -117,25 +117,36 @@ def ensure_column(self, column: str, column_type: str=None):
...
+            try:
+                self.athena.query(f'ALTER TABLE {self.table_name} ADD COLUMNS ({column} {column_type})')
+            except (self.athena.client.exceptions.ClientError, CidCritical) as exc:
+                raise CidCritical(f'Column {column} is not found in CUR and we were unable to add it. Please check FAQ.') from exc
+            self._metadata = self.athena.get_table_metadata(self.table_name) # refresh table metadata
+            logger.critical(f"Column '{column}' was added to CUR ({self.table_name}). Please make sure crawler do not override that columns. Crawler='{crawler_name}'")
+            return
+
+        # if table cannot be updated, check if it is ri/sp case - let's hope dashboard views can handle it:
+        if column in self.sp_required_columns + self.ri_required_columns:
```
Comment (yprikhodko) [inline cid/helpers/cur.py:143]: Why is this needed?

**Review by yprikhodko — APPROVED** [review]

File: cid/helpers/cur.py (hunk 1 above)
Comment (iakov-aws) [reply cid/helpers/cur.py:143]: Because we still can handle absence of RI / SP columns with custom SQL
